config/main.py

```diff
@@ -706,7 +706,7 @@ def _reset_failed_services():
 
 def _restart_services():
     click.echo("Restarting SONiC target ...")
-    clicommon.run_command("sudo systemctl restart sonic.target")
+    clicommon.run_command("sudo systemctl restart sonic.target --job-mode replace-irreversibly")
```

Comment:
@vivekreddynv please double check this change
@stepanblyschak please elaborate on the consequences

Reply:
@vivekreddynv It looks like if restart is executed with ```--job-mode replace-irreversibly```  we will still have the same issue, because the start job will be placed in the in systemd's job queue as "replace-irreverisbly" and the next ```config reload``` stop job will be discarded by systemd due to "replace-irreverisbly" of the start job leading to an error that looks smth like this:
```
Transaction for sonic.target/stop is destructive
```

Could you please double check? I think we only need to stop services with the guaranty that it will be successfully executed.

Reply:
when the restart job (and all the other dependent jobs) are finished and is taken out of systemd's job queue, i don't think  the next stop job will be cancelled. 

However, simultaneous config reloads in the quick succession, can lead to the behavior you have said. 

```
admin@sonic:~$ sudo systemctl restart sonic.target --job-mode replace-irreversibly
admin@sonic:~$ sudo systemctl stop sonic.target --job-mode replace-irreversibly (Ran Immediately)
Failed to stop sonic.target: Transaction for sonic.target/stop is destructive (ntp-config.service has 'start' job queued, but 'stop' is included in transaction).
See system logs and 'systemctl status sonic.target' for details.
(After all the dependent jobs of restart sonic.target  are done, it works)
admin@sonic:~$ sudo systemctl stop sonic.target --job-mode replace-irreversibly
admin@sonic:
```

The problem here is that the sonic.target start is not a blocking call unlike sonic.target stop. 

Nevertheless, the idea of this PR is that config reload should not fail and thus it makes sense to remove this. I'll raise a separate PR